spec/datadog/core/utils_spec.rb

```diff
@@ -1,7 +1,10 @@
 # typed: false
+require 'spec_helper'
```

Comment:
Minor: This is not needed -- our `.rspec` already includes `--require spec_helper` which is automatically added when running RSpec.